docroot/modules/custom/govcms_draft_content_access/govcms_draft_content_access.install

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * @file
+ * Contains govCMS Draft content access install module.
+ */
+
+use Drupal\user\Entity\User;
+
+/**
+ * Implements hook_install().
+ */
+function govcms_draft_content_access_install() {
+  // Create user on module install with Draft content access role.
+  $language = \Drupal::languageManager()->getCurrentLanguage()->getId();
+  $user = User::create();
+  $user_name = 'autologin';
+  $user_email = 'autologin@govcms.com.au';
+  $user_role = 'draft_content_access';
+  $user_password = uniqid();
+
+  // Mandatory fields.
+  $user->setPassword($user_password);
+  $user->enforceIsNew();
+  $user->setEmail($user_email);
+  $user->setUsername($user_name);
+
+  // Additional fields.
+  $user->set('init', 'email');
+  $user->set('langcode', $language);
+  $user->set('preferred_langcode', $language);
+  $user->set('preferred_admin_langcode', $language);
+  $user->addRole($user_role);
+  $user->activate();
+
+  // Save user account and log it.
+  $user_obj = $user->save();
+
+  if ($user_obj) {
+    \Drupal::logger('govcms_draft_content_access')->notice(
+      'Autologin user created with Username: @username, Email: @email, Password: @password, and Role: @role.', [
+        '@username' => $user_name,
+        '@email' => $user_email,
+        '@password' => $user_password,
+        '@role' => $user_role,
+      ]
+    );
+  }
+}
+
+/**
+ * Implements hook_schema().
+ */
+function govcms_draft_content_access_schema() {
```

Comment:
That would mean that tokens generated for content would be exportable, right?

It would make more sense to have it as a content entity but without the field UI; that way you get the Entity API but the tokens are not exportable.